tutorials/3d/standard_material_3d.rst

```diff
@@ -47,45 +48,56 @@ the materials unusable with many mid- and post-processing effects
 For this reason, materials in Godot are assumed opaque unless
 specified otherwise. The main settings that enable transparency are:
 
-* Transparent flag (this one)
+* Transparency (this one)
 * Blend mode set to other than "Mix"
 * Enabling distance or proximity fade
 
-Use Shadow to Opacity
-~~~~~~~~~~~~~~~~~~~~~
+When transparency other than ``0`` or ``1`` is not needed, it's possible to
+set a threshold to prevent the object from rendering semi-transparent pixels
+using the alpha scissor opetion.
 
-Lighting modifies the alpha so shadowed areas are opaque and non-shadowed
-areas are transparent. Useful for overlaying shadows onto a camera feed in AR.
+.. image:: img/spatial_material12.png
 
-Unshaded
-~~~~~~~~
+This renders the object via the opaque pipeline, which is faster and allows it
+to use mid- and post-process effects such as SSAO, SSR, etc.
 
-In most cases it is common for materials to be affected by lighting (shaded).
+Blend Mode
+~~~~~~~~~~
 
-However, in some cases you might want to show just the albedo (color) and
-ignore the rest. Toggling this flag on will remove all shading and display
-pure, unlit color.
+Controls the blend mode for the material. Keep in mind that any mode
+other than *Mix* forces the object to go through the transparent pipeline.
 
-.. image:: img/spatial_material26.png
+* **Mix:** Default blend mode, alpha controls how much the object is visible.
+* **Add:** Object is blended additively, nice for flares or some
```

Comment:
Maybe "the final color of the object is added to the color of the screen..."

And a similar phrase for the below two